invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/TitlesField/TitlesField.js

```diff
@@ -8,37 +8,52 @@
 
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-
-import { FieldLabel, TextField } from "react-invenio-forms";
+import Overridable from "react-overridable";
+import {
+  FieldLabel,
+  TextField,
+  createOverridableShowHideComponent,
+  mandatoryFieldCommonProps,
+} from "react-invenio-forms";
 import { AdditionalTitlesField } from "./AdditionalTitlesField";
 import { i18next } from "@translations/invenio_rdm_records/i18next";
 
-export class TitlesField extends Component {
+class TitlesFieldComponent extends Component {
   render() {
-    const { fieldPath, options, label, required, recordUI } = this.props;
+    const { fieldPath, options, label, labelIcon, recordUI, helpText, placeholder } =
+      this.props;
 
     return (
-      <>
-        <TextField
-          fieldPath={fieldPath}
-          label={<FieldLabel htmlFor={fieldPath} icon="book" label={label} />}
-          required={required}
-          className="title-field"
-          optimized
-        />
-        <AdditionalTitlesField
-          options={options}
-          recordUI={recordUI}
-          fieldPath="metadata.additional_titles"
-        />
-      </>
+      <Overridable
+        id="InvenioRdmRecords.DepositForm.TitlesField.Container"
+        optimized
```

Comment:
I saw you removed `optimized` in another example from the overridable props. Maybe do it also here and if it is applicable to other places?

Reply:
`optimized` is a prop that reasonably should be overridable. It controls whether [`FastField`](https://formik.org/docs/api/fastfield) instead of [`Field`](https://formik.org/docs/api/field) is rendered, which improves performance on large forms such as our deposit form but can also cause a lot of bugs, so implementors can definitely have some use cases where they want to disable this. I will look at implementing support for `optimized` overrides across the deposit form fields so we do it more consistently.